<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdytujPojazdDialog</class>
 <widget class="QDialog" name="EdytujPojazdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="kuferLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Czy ma kufer?</string>
   </property>
  </widget>
  <widget class="QLabel" name="rokLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmień rok</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="drzwiSpin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ladownoscLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>zmien ladownosc</string>
   </property>
  </widget>
  <widget class="QLabel" name="pojemnoscLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmień pojemność</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pojemnoscSpin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cenaLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmień cene</string>
   </property>
  </widget>
  <widget class="QLabel" name="przebiegLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmień przebieg</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cenaSpin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="rokSpin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="kuferCheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ladownoscSpin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="drzwiLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>zmien drzwi</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="przebiegSpin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
